server/middleware/setUpStaticResources.ts

```diff
@@ -22,9 +22,19 @@ export default function setUpStaticResources(): Router {
     '/node_modules/govuk-frontend/dist',
     '/node_modules/@ministryofjustice/frontend/moj/assets',
     '/node_modules/@ministryofjustice/frontend',
+    '/node_modules/jquery/dist',
   ).forEach(dir => {
     router.use('/assets', express.static(path.join(process.cwd(), dir), cacheControl))
   })
+  ;['/node_modules/jquery/dist/jquery.min.js'].forEach(dir => {
+    router.use('/assets/js/jquery.min.js', express.static(path.join(process.cwd(), dir), cacheControl))
+  })
+  ;['/node_modules/jquery-ui-dist/jquery-ui.min.js'].forEach(dir => {
+    router.use('/assets/js/jquery-ui.min.js', express.static(path.join(process.cwd(), dir), cacheControl))
+  })
+  ;['/node_modules/jquery-ui-dist/jquery-ui.min.css'].forEach(dir => {
+    router.use('/assets/stylesheets/jquery-ui.min.css', express.static(path.join(process.cwd(), dir), cacheControl))
```

Comment:
do we need jquery?

Reply:
Yes we need that for MOJFrontend.SortableTable and other MOJFrontend stuff.